source/heatfluxClass.py

```diff
@@ -554,6 +555,34 @@ def multiExp_profile_fluxspace(self, PFC, R, Bp, psiN, mode):
         q[nan_locations] = 0.0
         q[inf_locations] = 0.0
         return q
+    
+    def from_rzq_profile(self, rzq_data, PFC):
+        """"
+        q from any R(m), Z(m), q(W/m2) data
```

Comment:
is q W/m^2 or MW/m^2?

Reply:
input is (W/m2) and output will be MW/m2